webcommon/javascript2.editor/src/org/netbeans/modules/javascript2/editor/hints/ErrorCheckingSupport.java

```diff
@@ -86,42 +84,27 @@ public static void setErrorCheckingEnabledForMimetype(String mimeType, boolean e
         }
     }
 
-    public static String getMimeType(Parser.Result result) {
-        SyntaxAnalyzerResult syntax = getSyntaxAnalyzerResult(result);
-        if (syntax != null) {
-            return Utils.getWebPageMimeType(syntax);
+    public static String getMimeType(Parser.Result info) {
```

Comment:
@sdedic Could you please have a look at https://issues.apache.org/jira/browse/NETBEANS-5155